### PR TITLE
T347117 Repo Nuke Test flaky

### PR DESCRIPTION
**Pull request by rti**
https://phabricator.wikimedia.org/T347117

### CODE REVIEW

File: test/specs/repo/extensions/nuke.js [hunk 1]
```diff
@@ -46,27 +46,15 @@ describe( 'Nuke', function () {
...
-				timeoutMsg: 'Page should be deleted by now.'
-			}
+		result = await browser.makeRequest(
+			process.env.MW_SERVER + '/wiki/Vandalism',
+			{ validateStatus: false },
+			{}
 		);
 
-		assert.strictEqual( result.status, 404 );
+		return result.status === 404;
```
Comment (RickiJay-WMDE) [inline test/specs/repo/extensions/nuke.js:58]: Please assert rather than return, for consistency's sake

File: test/specs/repo/extensions/nuke.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ describe( 'Nuke', function () {
 			'Vandals In Motion'
 		);
 
-		const result = await browser.makeRequest(
+		let result = await browser.makeRequest(
```
Comment (RickiJay-WMDE) [inline test/specs/repo/extensions/nuke.js:21]: Can you make `result` two different variables, ideally with some indication of what each is the result of?

**Review by RickiJay-WMDE — CHANGES_REQUESTED** [review]

File: test/specs/repo/extensions/nuke.js (hunk 1 above)
Comment (rti) [reply test/specs/repo/extensions/nuke.js]: good catch, actually a copy pasta error. thanks. done

File: test/specs/repo/extensions/nuke.js (hunk 2 above)
Comment (rti) [reply test/specs/repo/extensions/nuke.js]: readability first. fully agree

Comment (rti) [reply test/specs/repo/extensions/nuke.js]: done

**Review by RickiJay-WMDE — APPROVED** [review]

**Pull request merged**